src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -176,11 +176,17 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
 
     val stoppedCluster = makeCluster(3).copy(status = ClusterStatus.Stopped).save()
 
+    val autopauseDisabledCluster = makeCluster(4).copy(
+      auditInfo = auditInfo.copy(dateAccessed = Instant.now().minus(100, ChronoUnit.DAYS)),
+      status = ClusterStatus.Running,
+      autopauseThreshold = 0).save()
+
     val autoFreezeList = dbFutureValue { _.clusterQuery.getClustersReadyToAutoFreeze() }
     autoFreezeList should contain (runningCluster1)
     //cluster2 is already stopped
     autoFreezeList should not contain stoppedCluster
     autoFreezeList should not contain runningCluster2
+    autoFreezeList should not contain autopauseDisabledCluster
```

Comment:
This check failed before the fix in `ClusterComponent`; now it passes.

Reply:
looks like the test only covered the case where `autopauseThreshold` is set to 0. Do we need to test the case where `autopause ` is `true`?

Reply:
The other clusters (`runningCluster1`, `runningCluster2`, `stoppedCluster`) all have `autopause` = `true` with a threshold of 30 mins. However it might be good to test other thresholds too. I'll add some more cases to the test.

Reply:
ah I see...I'm fine with not adding more tests in that case :)